### ISSUE TITLE
Empty module for some train stations

### ISSUE THREAD

**Issue by fabkai** [CFenner/MagicMirror-LocalTransport-Module#10]
Hello @CFenner ,

Your module is great, many thanks for that.

Unfortunately I have the problem that some connections are not displayed (not all!) at the mirror (after "load connections ..." it remains empty). But the train stops are located in Google Maps (API).

Do you have any idea?

As an example my current Config:
```
                {
                        module: 'localtransport',
                        header: 'Bahn nach Heddernheim',
                        position: 'top_left',
                        config: {
                                api_key: '***',
                                origin: 'Konstablerwache',
                                destination: 'Heddernheim',
                                updateInterval: 2,
                        }
                },
```

**Comment by alleslukas** [comment]
Same problem here. Any ideas?

**Comment by CFenner** [comment]
Hey there, please try to enter the names, that google suggests.

`Heddernheim, Frankfurt am Main`
`Konstablerwache, 60313 Frankfurt am Main`

Maybe that works, I'm not using the module currently, so I can't try it.

**Comment by Dom1n1c** [comment]
I also noticed it at smaller stations. but i will only show two working (bigger) destination stations since i found no solution for the smaller stations located in the city.
You can try to use the place_id from google, but that did not help me with my "small" station

**Comment by niklasbeinghaus** [comment]
Try using coordinates instead, i.e.:

for Humboldtstr. 2, Freiburg I used:
`47.99343529999999,7.848322100000001`
which works perfectly for now.

**Comment by fabkai** [comment]
Unfortunately no success with the place_id and the coordinates. For lager stations, everything works. Small subway and tram stations don’t work.
@CFenner Have you another idea or you could install the module again?

**Comment by fabkai** [comment]
I have again tested different subway / tram stations in different ways, for example:
origin: 
Coordinates: '50.154956, 8.686294',
PlaceID: 'ChIJg4KbNSwPvUcRG6T13dfeYy4', 
Text: 'Sigmund-Freud-Straße, Frankfurt am Main, Deutschland',

destination: 
Coordinates: '50.082594, 8.588766', 
PlaceID: 'ChIJtSOq0UQKvUcR8peq8_WsWAI', 
Text: 'Ferdinand-Dirichs-Weg, Frankfurt am Main, Deutschland',

Unfortunately without success. The Google Developer Console displays the requests without errors. But the mirror remains empty. In the log of the mirror is also nothing to see.

What can this still be?

**Comment by GHLasse** [comment]
Google Maps tells me 
Die Route von "Sigmund-Freud-Straße, 60435 Frankfurt am Main" nach "Ferdinand-Dirichs-Weg, 60529 Frankfurt am Main" mit öffentlichen Verkehrsmitteln konnte nicht berechnet werden.
or
Die Route von "50.154956, 8.686294" nach "50.082594, 8.588766" mit öffentlichen Verkehrsmitteln konnte nicht berechnet werden.
By the way, Google Maps shows me where these places are and returns a 3 hour walk if asked for it. Which is odd, but at least explains where the problem lies. (i.e. with Google/ the local transport provider)

Maybe one can display an error message instead of an empty list, though the empty list so far seems to be the nicer solution, as sometimes during the night some routes are not available and you don't always want an error message for that.

**Comment by fabkai** [comment]
@GHLasse Yes you are right. I have not tested it on Google Maps directly =)
Apparently, maps do not calculate the travel time between two metro stations in Frankfurt.

As an example:
Start metro station: Sigmund-Freud-Straße
Destination metro station: Ronneburg-Straße
Despite the fact that I have chosen public transport, Google Maps suggests a route afoot:
https://www.google.de/maps/dir/Sigmund-Freud-Stra%C3%9Fe,+60435+Frankfurt+am+Main/Ronneburgstra%C3%9Fe,+Frankfurt+am+Main/@50.152709,8.6853032,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x47bd0f2c359b8283:0x2e63ded7ddf5a41b!2m2!1d8.686277!2d50.1547225!1m5!1m1!1s0x47bd0f29140d240b:0xfc941fd8a6bbd087!2m2!1d8.6886179!2d50.15068!3e3

Probably this is the fault.

**Comment by GHLasse** [comment]
So Google didn't consider Frankfurt am Main important enough to add the public transport routing for it? Very strange but if this is the case, the only option I can think of, is to at least display the walking option. But that requires another request in case the first one returns empty... I guess people living in Frankfurt a. M. will have to rely on other localtransport modules :-( or just complain to Google to fix it :-)
